docs/pbs/advanced-pbs-topics.md

```diff
@@ -0,0 +1,460 @@
+# Advanced PBS Topics
+
+## Job Dependencies
+
+It is possible to schedule jobs to run based on the status of other
+jobs. For example, you might schedule a preprocessing job to run;
+start a computation job when the preprocessing job is complete; then
+start a post-processing job when the computation is done.
+
+One way to schedule such a series or chain of jobs is to use `qsub -W [job-dependency-expression]`
+to specify the job dependencies you need.  This can also be accomplished by submitting subsequent jobs from *inside* another job.
+
+### PBS job dependencies (`-W depend`)
+Let's say you have you have three scripts to submit and run consecutively:
+
+ 1. `pre.pbs`: a preprocessing job
+ 2. `main.pbs`: a computation job
+ 3. `post.pbs`: a post-processing job
+
+The main job can be run only when the preprocessing job finishes, and the post-processing job can be run only when the computation job finishes.
+
+Submit the first job, placing it on hold. (If it starts before the dependent jobs are submitted, the dependent jobs might never start.):
```

Comment:
I don't think this *hold the job first* contrivance is required any longer, but I can check with Jenett to be sure.